packages/arb-token-bridge-ui/src/components/TransferPanel/TokenInfoTooltip.tsx

```diff
@@ -0,0 +1,93 @@
+import { ERC20BridgeToken } from '../../hooks/arbTokenBridge.types'
+import { useNetworks } from '../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+import { shortenAddress } from '../../util/CommonUtils'
+import { captureSentryErrorWithExtraData } from '../../util/SentryUtils'
+import { ExternalLink } from '../common/ExternalLink'
+import { Tooltip } from '../common/Tooltip'
+import { TokenLogo } from './TokenLogo'
+import { useNativeCurrency } from '../../hooks/useNativeCurrency'
+
+const createBlockExplorerUrlForToken = ({
```

Comment:
Moved from old file `TokenSymbolWithExplorer`

Reply:
we also have `BlockExplorerTokenLink` in `TokenRow`, maybe consolidate them into a util?

doing it in a future ticket is fine